extension/web-ext-config.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ module.exports = {
+  // Global options:
+  sourceDir: ".",
+  artifactsDir: "../static/downloads/addon/latest/",
```

Comment:
I actually want to stop building the artifacts into the static downloads folder. We never ended up using it.
```suggestion
```